src/types/Feed.ts

```diff
@@ -34,6 +34,7 @@ export type FeedValue = number | HexStringWith0x;
 
 export interface FeedFetcher {
   name: FetcherName;
+  symbol?: string;
```

Comment:
is this in use in this PR?

Reply:
no, good catch! I think I used it in the initial implementation and then I changed the way.